specification/cognitiveservices/data-plane/Content Moderator/v1.0/contentmoderatorImageText.swagger.json

```diff
@@ -0,0 +1,591 @@
+          "image/png",
+          "image/bmp",
+          "image/tiff"
+        ],
+        "responses": {
+          "200": {
+            "description": "The found text and list of candidate text.",
+            "schema": {
+              "$ref": "#/definitions/OCRResponse"
+            }
+          },
+          "default": {
+            "description": ""
+          }
+
+        },
+        "produces": [
+          "application/json"
+        ]
+      }
+    },
+    "/ProcessImage/Evaluate": {
+      "post": {
+        "tags": [
+          "Images"
+        ],
+        "operationId": "Images_Evaluate",
+        "description": "Returns probabilities of the image containing racy or adult content.",
+        "parameters": [
+          {
+            "name": "CacheImage",
+            "in": "query",
+            "description": "Whether to retain the submitted image for future use; defaults to false if omitted",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
+          },
+          {
+            "$ref": "#/parameters/contentTypeHeader4"
+          },
+          {
+            "name": "file",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "file"
+            },
+            "description": "The file location."
+          }
+        ],
+        "consumes": [
+          "image/gif",
+          "image/jpeg",
+          "image/png",
+          "image/bmp",
+          "image/tiff"
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/EvaluateResponse"
+            }
+          },
+          "default": {
+            "description": ""
+          }
+        },
+        "produces": [
+          "application/json"
+        ]
+      }
+    },
+    "/ProcessImage/Match": {
+      "post": {
+        "tags": [
+          "Images"
+        ],
+        "operationId": "Images_Match",
+        "description": "Fuzzily match an image against one of your custom Image Lists. You can create and manage your custom image lists using <a href=\"/docs/services/578ff44d2703741568569ab9/operations/578ff7b12703741568569abe\">this</a> API. \r\n\r\nReturns ID and tags of matching image.<br/>\r\n<br/>\r\nNote: Refresh Index must be run on the corresponding Image List before additions and removals are reflected in the response.",
+        "parameters": [
+          {
+            "name": "listId",
+            "in": "query",
+            "description": "The list id.",
+            "type": "string"
+          },
+          {
+            "name": "cacheimage",
+            "in": "query",
+            "description": "Use cached image.",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
```

Comment:
CI seems to be failing due to the duplicate parameters. 

https://travis-ci.org/Azure/azure-rest-api-specs/jobs/264921311#L3501